distribution/docs/src/main/resources/_contents/_securing/managing-certificates-contents.adoc

```diff
@@ -0,0 +1,357 @@
+
+==== Certificate Management
+
+${branding} uses certificates in two ways:
+
+["lowerroman"]
+. Transmitting and receiving encrypted messages.
+. Authenticating an incoming user request.
+
+For more certificate configuration options, see <<_configuring_${ddf-security}, Configuring ${ddf-security}>>.
+
+===== Keystores
+
+To manage certificates and keys, ${branding} includes a default keystore.
+
+.Default Keystore Files
+[cols="5m" options="header"]
+|===
+
+|Host
+|Keystore
+|Truststore
+|Configuration Location
+|Usage
+
+|localhost
+|serverKeystore.jks
+|serverTruststore.jks
+|etc/org.ops4j.pax.web.cfg
+
+etc/system.properties
+
+etc/ws-security/server/encryption.properties
+
+etc/ws-security/server/signature.properties
+
+etc/ws-security/issuer/encryption.properties
+
+etc/ws-security/issuer/signature.properties
+
+etc/system.properties
+|Used to secure (SSL) all of the endpoints for ${branding}, to perform outgoing SSL requests, and sign STS SAML assertions.
+This also includes the ${admin-console} and any other web service that is hosted by ${branding}.
+
+|===
+
+[NOTE]
+====
+The following information was sourced from https://www.racf.bnl.gov/terapaths/software/the-terapaths-api/example-java-client/java-client/setting-up-keystores-with-jetty-and-keytool.
+====
+
+====== Creating a Client Keystore
+
+[WARNING]
+====
+This walk-through details how to use a PKCS12 store. This is the most popular format used when exporting from a web browser.
+====
+
+. Obtain a personal ECA cert (client certificate)
+.. To do this, open *Internet Explorer -> Tools -> Options*.
+.. Select the *Content* tab.
+.. Click *Certificates*
+.. Select the *Personal* tab.
+.. Select the _certificate_ needed to export. There should be the one without a "Friendly Name" and it is not the "Encryption Cert").
+.. Click *Export*.
+.. Follow *Certificate Export Wizard*.
+.. When a prompt requests to export the private key, select *Yes*.
+. Add a cert to a new java keystore, replacing cert with the name of the PKCS12 Keystore needed to convert, and replace MY_KEYSTORE.jks with the desired name of the Java Keystore:
```

Comment:
Line 68 is not clear. Delete it and replace with: 

Internet Explorer will export the certificates as a PKCS 12 file. This file can imported into ${branding}'s server keystore using the steps in "Keystore Management"